### PR TITLE
Radiobuttons and selects values that are not strings

### PR DESCRIPTION
**Pull request by ksenia-zalesnaya**
Extracted from other [PR](https://github.com/hanami/helpers/pull/73). Tests state the problem. I don't know whether `to_s` is best solution, anyone has any ideas?

### CODE REVIEW

**Comment by jodosha** [comment]
@TiteiKo Hi. Both this PR and the other one by @ksenia-zalesnaya (#73) have git conflicts because we merged your #81.

Would you be so kind to review the code and make sure that both the fixes are merged? Thank you all! 😄

File: test/form_helper_test.rb [hunk 1]
```diff
@@ -1118,16 +1118,32 @@
...
+            radio_button :category, 'Non-Fiction'
+          end.to_s
+
+          actual.must_include %(<input type="radio" name="book[category]" value="Fiction">\n<input type="radio" name="book[category]" value="Non-Fiction" checked="checked">)
+        end
+      end
+
+      describe 'decimal value' do
+        let(:params) { Hash[book: { price: val }] }
+        let(:val)    { '20.0' }
```
Comment (TiteiKo) [inline test/form_helper_test.rb]: To test on a decimal value, shouldn't the val be `20.0` and not `'20.0'`?

File: test/form_helper_test.rb [hunk 2]
```diff
@@ -1256,15 +1272,31 @@
...
 
-          actual.must_include %(<select name="book[store]" id="book-store">\n<option>Select a store</option>\n<option value="it" selected="selected">Italy</option>\n<option value="us">United States</option>\n</select>)
+            actual.must_include %(<select name="book[store]" id="book-store">\n<option>Select a store</option>\n<option value="it" selected="selected">Italy</option>\n<option value="us">United States</option>\n</select>)
+          end
+        end
+
+        describe 'integer values' do
+          let(:values) { Hash['Brave new world' => 1, 'Solaris' => 2] }
+          let(:params) { Hash[bookshelf: { book: val }] }
+          let(:val)    { '1' }
```
Comment (TiteiKo) [inline test/form_helper_test.rb]: Same question as above

File: test/form_helper_test.rb (hunk 1 above)
Comment (TiteiKo) [inline test/form_helper_test.rb]: As the params sent to the form are the one before cast, this PR is needed :)

Comment (ksenia-zalesnaya) [inline test/form_helper_test.rb]: So they should be strings, isn't it?

**Comment by jodosha** [comment]
@ksenia-zalesnaya @TiteiKo is this PR ready? Can we merge it? ✨  😄

File: test/form_helper_test.rb (hunk 1 above)
Comment (TiteiKo) [inline test/form_helper_test.rb]: They can either be string or others, so the test is fine 👍

**Comment by TiteiKo** [comment]
@ksenia-zalesnaya Thanks a lot for this PR ⭐️

**Pull request merged**